### ISSUE TITLE
Sugestões de features

### ISSUE THREAD

**Issue by brayann** [laravelbrasil/laravel.com.br#2]
Abrindo um espaço para discussão sobre o que devemos incluir na plataforma

**Comment by meredyk** [comment]
Para iniciar a discussão:

- Novidades do Laravel
- Dicas
- Vagas de emprego
- Divulgação de cursos e materiais para estudo
- Eventos

**Comment by brayann** [comment]
> Para iniciar a discussão:
> 
> * Novidades do Laravel
> * Dicas
> * Vagas de emprego
> * Divulgação de cursos e materiais para estudo
> * Eventos

Boa, porém acho que isso cairia na questão de quais categorias ter. Essa thread é para definir features, p.e. ter filtro por categorias, que seriam essas q vc sugeriu :)

**Comment by robersonfaria** [comment]
Então, vou pensar em features, mas acho que poderíamos estabelecer algumas boas práticas a serem seguidas nesse desenvolvimento tipo, usar interfaces, services, solid, regras claras de nomeação de classes e métodos.

Esse projeto por si só já pode ser algo pra colaborar e muito com a comunidade, podemos fazer code review. Contém comigo.

Vou pensar em features, mas acho q pra termos algo rápido e depois evoluir, poderia pensar em um blog simples. Posts com categorias e tags. Depois vamos adicionado novas features e áreas no site.

Ótima iniciativa pessoal, novamente, contém comigo

**Comment by brayann** [comment]
> Então, vou pensar em features, mas acho que poderíamos estabelecer algumas boas práticas a serem seguidas nesse desenvolvimento tipo, usar interfaces, services, solid, regras claras de nomeação de classes e métodos.
> 
> Esse projeto por si só já pode ser algo pra colaborar e muito com a comunidade, podemos fazer code review. Contém comigo.
> 
> Vou pensar em features, mas acho q pra termos algo rápido e depois evoluir, poderia pensar em um blog simples. Posts com categorias e tags. Depois vamos adicionado novas features e áreas no site.
> 
> Ótima iniciativa pessoal, novamente, contém comigo

Concordo em tudo, na real tava pensando em criar um blog no Medium nesse meio tempo

**Comment by meredyk** [comment]
> > Para iniciar a discussão:
> > 
> > * Novidades do Laravel
> > * Dicas
> > * Vagas de emprego
> > * Divulgação de cursos e materiais para estudo
> > * Eventos
> 
> Boa, porém acho que isso cairia na questão de quais categorias ter. Essa thread é para definir features, p.e. ter filtro por categorias, que seriam essas q vc sugeriu :)

Ahh, desculpa rsrs, é que eu levo a risca o significado de feature em um projeto, que seria uma funcionalidade do sistema que entrega um benefício ou resolve um problema real do cliente, geralmente representada através de User Stories (nos métodos ágeis).

**Comment by robersonfaria** [comment]
@brayann acredito que essa documentação de padrões e tudo mais relacionado ao projeto pode ser uma pasta docs com arquivos markdown aqui no próprio projeto Git. O medium serveria pra ir "contando" como tá se desenrolando o projeto e chamando mais pessoas a participar.

@meredyk com certeza suas ideias de features serão aproveitadas kkkk mas pensei na ideia de blog simples pra iniciar meio que como um MVP e ter algo pra relatar a própria evolução do projeto o quanto antes.

**Comment by Tagliatti** [comment]
Todo mundo aqui deve conhecer o https://laravel-news.com/, pq não fazemos algo em português no mesmo sentido? Inclusive podemos traduzir alguns tutoriais de lá mesmo, isso iria enriquecer muito o assunto em pt-br.

**Comment by robersonfaria** [comment]
Será que poderíamos utilizar o mesmo layout do laravel-news? já ajudaria bastante a agilizar o processo, caso agente não possa utilizar dai teria que encontrar alguém bom pra fazer algo legal, eu sou péssimo nisso kkkk

**Comment by ElderCastroLima** [comment]
Isso mesmo começar com algo simples seria uma boa ideia. Começar com um blog ir evoluindo um MVP até chegar a versão final.

**Comment by Tagliatti** [comment]
Vou ver esse lance do laravel news

**Comment by jakjr** [comment]
Ótima iniciativa! Contem comigo......

**Comment by Dhayllin** [comment]
Galera as ideiais são boas, tem de ser definido o cronograma para o projete ser implementado. As pegar as melhores sugestões e sentar a macha.

**Comment by a21ns1g4ts** [comment]
Eu acho que deveria ter uma seção para exibir projetos brasileiros em laravel. Um showcase de projetos nacionais.

**Comment by robersonfaria** [comment]
Bom pessoal, acho que pra dar essa agilidade precisamos definir como serão tomadas as decisões, algum esquema de votação ou de aprovação de ideias.

Eu sou a favor de escrever a ferramenta conforme regras estipuladas por nós mesmo, mas temos alguns colegas que sugerirão uso de algum CMS ou ferramente de BLOG pronta, como vamos definir e começar a desenvolver?

Podemos usar os "likes" dos comentários mesmo, tipo.

Reajam a esse comentário com:
:rocket:  para desenvolver uma ferramenta do zero
:eyes:  para utilizar uma ferramenta pré existente

Dai estipulamos um prazo para essa votação ficar em aberta de tipo 24horas e aguardamos o resultado.

O que acham? Ou alguém tem outra sugestão?
Podemos também criar um form no google forms e capturar a opinião do pessoal, mas antes teríamos que montar uma base de emails com os interessados.

Como fazemos?

**Comment by Tagliatti** [comment]
Não acho que seja o momento de ver a plataforma, independente se vai ser WordPress, Laravel, Jekyll... Precisamos de um layout, então vamos correr atrás de achar um layout legal. 🏃‍♂️🏃‍♀️

**Comment by robersonfaria** [comment]
Temos bastante gente interessada em participar, podemos correr com as duas coisas em paralelo, enquanto alguém corre com o layout outras pessoas podem ir desenvolvendo features, basta antes definirmos o que será feito.

**Comment by robersonfaria** [comment]
O que acham desse formulário? Fiz algumas perguntas básicas que acho que já dão um norte para o desenvolvimento.
Se quiserem adicionar questões me avisem que vou adicionando e na segunda que vem se ninguém se opor ao envio do formulário, gero o link publico e disponibilizo aqui para ser respondido.

![image](https://user-images.githubusercontent.com/5167382/57560238-d7211b00-735b-11e9-97ff-740de75f3d44.png)

Logicamente depois disponibilizo de forma aberta os dados para todos os interessados.

Pensei em pedir o nome e email de quem ta preenchendo no início para contactarmos quando precisarmos de alguma habilidade da pessoa. Mas não tenho certeza se solicito ou não, o que acham?

Valeu

**Comment by meredyk** [comment]
@robersonfaria ficou top o formulário, bem direto ao ponto. Sobre pedir o nome/email, na minha opinião poderia ser em uma segunda etapa com base nos resultados desse formulário.

**Comment by jeanmalves** [comment]
Muitas idéias interessantes, mas acho que precisamos planejar o que queremos no site completo e depois realizar as ações em etapas porque muito provável de criarmos um frank.

**Comment by danganf** [comment]
Galera, boa tarde! Como faço para participar desse projeto? Curti bastante a ideia e acredito ter conhecimento para agregar ao projeto.

**Comment by Atiladanvi** [comment]
Alguma perspetiva para o início da implementação ?
